dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/AbsLimRateLimFeedthroughFreezeLimDetection.mo

```diff
@@ -27,9 +27,9 @@ block AbsLimRateLimFeedthroughFreezeLimDetection "First order feed-through with
     Placement(visible = true, transformation(origin = {-50, 40}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
   Modelica.Blocks.Logical.LessThreshold lessThreshold(threshold = YMin) annotation(
     Placement(visible = true, transformation(origin = {-50, -40}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
-  Dynawo.NonElectrical.Blocks.NonLinear.FixedBooleanDelay fixedBooleanDelay(Y0 = false, tDelay = tS) annotation(
+  Dynawo.NonElectrical.Blocks.NonLinear.FixedBooleanDelay fixedBooleanDelay(Y0 = Y0 > YMax, tDelay = tS) annotation(
```

Comment:
I anticipate that having a U0 instead of a Y0 could be better (Y0 = U0 > YMax), what do you think @cossartque ?

Reply:
Yes of course, U0, not Y0